### PR TITLE
Update Checkbox component

### LINKED ISSUE #122: Create <Checkbox /> component
## Description
Use a list of checkboxes when using a list with more than one selectable option. If you want to allow select just one option, use `<RadioButton />` instead.

## Note
Tbh, this component is basically just a variation of `<RadioButton />` component, only with different tokens. It should maybe be only one component with 2 types.

## Visual style (for radio and checkboxes)
radio-button--checkbox

Zeplin: https://zpl.io/bPdzpPA

### Interactions
_All same as for radio buttons_

### Additional information
_All same as for radio buttons_

## Functional specs
_All same as for radio buttons_

## Storybook chapters
One chapter for basic checkbox (is it default?), the second one for a checkbox with help text, playground. More after the knobs are defined.

## Some inspiration
http://elemental-ui.com/forms (scroll down to Checkbox)
https://mineral-ui.com/components/checkbox
https://polaris.shopify.com/components/forms/checkbox

### PR DESCRIPTION
**Pull request by wEprous**
This Pull Request meets the following criteria:

* [x] Tests have been added/adjusted for my new feature
* [x] New Components are registered in index.js of my project

Closes #122

**Plz ignore that some css properties don't have tokens now, need to add and update.**

### CODE REVIEW

File: src/Checkbox/index.js [hunk 1]
```diff
@@ -1,58 +1,117 @@
...
-      <div className="checkbox">
-        {props.checked && (
-          <div className="checked">
-            <Icon.Checked />
-          </div>
-        )}
-      </div>
-      <Typography variant={props.checked ? "bold" : "normal"}>{props.label}</Typography>
+const CheckBox = (props: Props) => {
+  const { label, name, value, error, disabled, checked, onChange } = props;
```
Comment (jaroslav-kubicek) [inline src/Checkbox/index.js]: that's too much, just using props should be better

**Comment by jaroslav-kubicek** [comment]
In order to close it finally, I rebased this PR, pls check it as well @wEprous so it can be merged

**Comment by wEprous** [comment]
plz *DO NOT* merge

**Comment by wEprous** [comment]
@tomashapl @jaroslav-kubicek plz CR once more

File: src/Checkbox/Checkbox.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+
+import { Check } from "../icons";
```
Comment (tomashapl) [inline src/Checkbox/Checkbox.js]: Can we import Check icon like this:

```
import Check from "../icons/Check";
```

File: src/Checkbox/Checkbox.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+const TextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-left: ${({ theme }) => theme.spaceXSmall};
+`;
+
+const Info = styled.span`
+  font-size: ${({ theme }) => theme.fontSizeMessageForm};
+  color: ${({ theme }) => theme.paletteInkLight};
```
Comment (tomashapl) [inline src/Checkbox/Checkbox.js]: Please add TODO and create token 🙏

File: src/Checkbox/__tests__/index.test.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import CheckBox from "../Checkbox";
+
+const label = "Checkbox";
+const onChange = jest.fn();
+const name = "checkbox";
+const values = ["option", ""];
+const disable = [true, false];
+
+values.map(value =>
+  disable.map(disabled =>
```
Comment (tomashapl) [inline src/Checkbox/__tests__/index.test.js]: Please use just one iteration of test 🙏

File: src/Checkbox/Checkbox.js [hunk 5]
```diff
@@ -0,0 +1,142 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+
+import { Check } from "../icons";
+
+type Props = {
```
Comment (tomashapl) [inline src/Checkbox/Checkbox.js]: please use strict types: `|`

File: src/Checkbox/Checkbox.js [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+import Check from "../icons/Check";
+
+type Props = {|
+  label: string,
+  value?: string,
+  hasError: boolean,
+  disabled: boolean,
+  checked: boolean,
+  info?: React.Node,
+  onChange: (SyntheticEvent<HTMLInputElement>) => any,
```
Comment (tomashapl) [inline src/Checkbox/Checkbox.js]: onChange should return nothing = `void`

**Comment by wEprous** [comment]
@tomashapl plz, delete branch after merge 🙏

**Pull request merged**